tests/data/schema/target_output/multityped.md

```diff
@@ -0,0 +1,74 @@
+# AWS::Color::Red
+
+a test schema
+
+## Syntax
+
+To declare this entity in your AWS CloudFormation template, use the following syntax:
+
+### JSON
+
+<pre>
+{
+    "Type" : "AWS::Color::Red",
+    "Properties" : {
+        "<a href="#property1" title="property1">property1</a>" : <i>String, Double, Map, [ <a href="obj2def.md">obj2def</a>, ... ]</i>,
+        "<a href="#obj1" title="obj1">obj1</a>" : <i><a href="obj2def.md">obj2def</a></i>,
+        "<a href="#arr1" title="arr1">arr1</a>" : <i>[ <a href="obj2def.md">obj2def</a>, ... ]</i>
+    }
+}
+</pre>
+
+### YAML
+
+<pre>
+Type: AWS::Color::Red
+Properties:
+    <a href="#property1" title="property1">property1</a>: <i>String, Double, Map,
+      - <a href="obj2def.md">obj2def</a></i>
```

Comment:
This will probably turn out a little funky if the list of objects doesn't come last in the list of types. Just something to consider

Reply:
done:
when it's a multitype we switch to json style - `[ Type, ... ]` if it's single type we keep `\n - Type`